Makefile

```diff
@@ -209,6 +209,22 @@ node: curl gpg
 		sudo apt-get install -qqy nodejs; \
 	fi
 
+gofmt:
+	if [[ "$$(which gofmt)" != "$(GOPATH)/bin/gofmt" ]]; then \
+		TMP_DIR=$$(mktemp -d); \
+		pushd $$TMP_DIR > /dev/null 2>&1; \
+		git clone "https://github.com/golang/go.git"; \
+		cd go; \
+		git checkout "origin/release-branch.go1.11"; \
+		mv "src/cmd" "$(GOPATH)/src/cmd"; \
```

Comment:
Do we have to move `src/cmd` to `GOPATH`? Can we build it inside the temp directory?